src/views/Home/Home.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { changeToDo, createTodo, fetchToDo } from '../../services/fetchutils';
+import './Home.css';
+
+
+export default function Home() {
+  const [todo, setToDo] = useState([]);
+  const [error, setError] = useState('');
+  const [description, setDescription] = useState('');
+ 
+
+  useEffect(()=> {
+    const grabToDo = async () => {
+      const data = await fetchToDo();
+      setToDo(data);
+    };
+    grabToDo();
+  }, []);
+
+  const submitToDo = async () => {
+   
+    try {
+      const data = await createTodo({ description });
+      setToDo((prevState) => [...prevState, data]);
+      
+    } catch (e) {
+      setError('you broke it');
+    }
+  };
+
+  const setToTrue = async (data) => {
+    try {
+      await changeToDo({ ...data, complete: true });
+      const checkbox = await fetchToDo();
+      setToDo(checkbox);
+    } catch (e) {
+      setError('tisk tisk you broke it');
+    }
```

Comment:
If you want the user to be able to toggle between complete and not complete, we can call await changeTodo({ ...data, complete: !data.complete}). This will change complete to be the opposite of what it's current value is.